app/assets/stylesheets/orders.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the orders controller here.
```

Comment:
Keeping these style sheets here for now, but we don't _really_ need them. Could delete.